{{cookiecutter.project_name}}/app/core/config.py

```diff
@@ -51,47 +52,49 @@ class Settings(BaseSettings):
     DEFAULT_DATABASE_HOSTNAME: str
     DEFAULT_DATABASE_USER: str
     DEFAULT_DATABASE_PASSWORD: str
-    DEFAULT_DATABASE_PORT: str
+    DEFAULT_DATABASE_PORT: int
     DEFAULT_DATABASE_DB: str
     DEFAULT_SQLALCHEMY_DATABASE_URI: str = ""
 
     # POSTGRESQL TEST DATABASE
     TEST_DATABASE_HOSTNAME: str = "postgres"
     TEST_DATABASE_USER: str = "postgres"
     TEST_DATABASE_PASSWORD: str = "postgres"
-    TEST_DATABASE_PORT: str = "5432"
+    TEST_DATABASE_PORT: int = 5432
     TEST_DATABASE_DB: str = "postgres"
     TEST_SQLALCHEMY_DATABASE_URI: str = ""
 
     # FIRST SUPERUSER
     FIRST_SUPERUSER_EMAIL: EmailStr
     FIRST_SUPERUSER_PASSWORD: str
 
-    @validator("DEFAULT_SQLALCHEMY_DATABASE_URI")
-    def _assemble_default_db_connection(cls, v: str, values: dict[str, str]) -> str:
-        return PostgresDsn.build(
-            scheme="postgresql+asyncpg",
-            user=values["DEFAULT_DATABASE_USER"],
-            password=values["DEFAULT_DATABASE_PASSWORD"],
-            host=values["DEFAULT_DATABASE_HOSTNAME"],
-            port=values["DEFAULT_DATABASE_PORT"],
-            path=f"/{values['DEFAULT_DATABASE_DB']}",
+    @model_validator(mode="after")
```

Comment:
This probably should be @computed_field as of pydantic 2, model validators are now meant more for validation, not for calculating values (this is somewhat a hack used, because pydantic 1 hasn't computed fields unfortunately)